projects/geoengine-app/src/app/app.component.scss

```diff
@@ -14,8 +14,9 @@ wave-layer-list {
 }
 
 wave-layer-list .logo .mat-icon {
-    width: 100%;
-    height: 100%;
+    font-size: 50px;
+    width: 150px;
```

Comment:
We shouldn't use pixel here, since it changes depending on the screen size. Please use % or `rem`.

Reply:
b1cfcea8d49faf88a55ebbbf1c37bfeee173192e